client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -106,16 +102,22 @@ def make_tensor_proto(values, dtype=None, shape=None):
     '''
     if shape is None:
         infered_shape = []
-    else:
+    elif _is_array_like(shape):
+        if shape != list(np.array(values).shape):
+            raise ValueError(f'shape provided does not match values array shape')
```

Comment:
Please check with `make_tensor_proto` from TF - in case we provide data in different shape than the shape from parameter what should we do. Before your adjustment we did not care about provided shape. Now we raise an error if it does not match. What TF does I think is that they reshape the data to expected shape (from parameter). @dtrawins  what is the desired behavior here?

Reply:
You are right, TF does indeed reshape the data to expected shape. In that case I think we might want to remove this.